jobs/integration/utils.py

```diff
@@ -418,32 +420,31 @@ async def validate_storage_class(model, sc_name, test_name):
             (control_plane, "po", [f"{sc_name}-write-test"]),
             timeout_msg=f"Unable to create write pod for {test_name} test",
         )
-
         # read that string from pvc
         pod_definition = f"""
 kind: Pod
 apiVersion: v1
 metadata:
   name: {sc_name}-read-test
 spec:
+  containers:
+    - name: {sc_name}-reader
+      image: rocks.canonical.com/cdk/busybox:1.36
+      command: ["/bin/cat"]
+      args: ["/mnt/data/hello.txt"]
+      volumeMounts:
+      - name: shared-data
+        mountPath: /mnt/data
   volumes:
   - name: shared-data
     persistentVolumeClaim:
       claimName: {sc_name}-pvc
-      readOnly: false
```

Comment:
this probably could be `readOnly: true` don't you think if it's true this pod won't write to the volume